Adapter.C

```diff
@@ -310,8 +310,8 @@ void preciceAdapter::Adapter::configure()
                 }
                 else if (inModules > 1)
                 {
-                    adapterInfo("It looks like more than one modules can read " + dataName
-                                    + "and I don't know how to choose. Try disabling one of the modules.",
+                    adapterInfo("It looks like more than one modules can read \"" + dataName
+                                    + "\" and I don't know how to choose. Try disabling one of the modules.",
```

Comment:
Did you try this locally? I think quotation marks are added automatically.

Reply:
They are not added automatically. Here is an example error message with the hard-coded quotes:

```
--> FOAM Warning : 
    From void adapterInfo(std::string, std::string)
    in file Utilities.C at line 50
    Error (deferred - will exit later) in the preCICE adapter: 
I don't know how to read "Fisplacement". Maybe this is a typo or maybe you need to enable some adapter module?
```

and one without the hard-coded quotes:

```
--> FOAM Warning : 
    From void adapterInfo(std::string, std::string)
    in file Utilities.C at line 50
    Error (deferred - will exit later) in the preCICE adapter: 
I don't know how to read Fisplacement. Maybe this is a typo or maybe you need to enable some adapter module?
```